pkg/server/config/config.go

```diff
@@ -0,0 +1,25 @@
+package config
+
+// Opts for individual xDS implementations that can be
+// utilized through the functional opts pattern
+type Opts struct {
+	// If true respond to ADS requests with a guaranteed resource ordering
+	Ordered bool
```

Comment:
TBH, I haven't looked at ordering for a long time. My recollection was that for ADS, the client was supposed to make requests in the order that was correct for it? If that's the case, do we really need to order anything in the server?

My other query is that, if ordering is a correctess issue, then shouldn't we be correct by default?

Reply:
That's what I thought, the bug this closes out shows that envoy sometimes requests things out of order even in ADS. 

IMO for ADS we always want to be ordered so yes. If I recall correctly when a user configures the server to run in ADS mode it'll be ordered by default (but you can override that).